### PR TITLE
Add initiate_chat parameter in documentation

### PR DESCRIPTION
**Pull request by saga-hs**
+ Add initiate_chat parameter in console qrcode docs
+ Add notes about it.

### CODE REVIEW

**Review by oitihjya-hs — APPROVED** [review]

File: docs/pc-and-console/helpshift-apis.mdx [hunk 1]
```diff
@@ -199,6 +199,17 @@ What extra information should be included in the response. A JSON array of strin
...
 
+</td>
+</tr>
+<tr>
+<td>initiate_chat</td>
+<td>No</td>
+<td>
+Whether to start a new chat when the previous issue is resolved. Valid values -
+
+- true:  a new chat will be started automatically when the previous issue is resolved and without needing any user action like clicking the New Conversation button or going through the post resolution flow like feedback bot for the previously resolved issue.
```
Comment (rohan-helpshift) [inline docs/pc-and-console/helpshift-apis.mdx:210]: A new chat

File: docs/pc-and-console/helpshift-apis.mdx [hunk 2]
```diff
@@ -199,6 +199,17 @@ What extra information should be included in the response. A JSON array of strin
...
+- false: a new chat will not be automatically started.
```
Comment (rohan-helpshift) [inline docs/pc-and-console/helpshift-apis.mdx:211]: A new chat

File: docs/pc-and-console/helpshift-apis.mdx (hunk 1 above)
Comment (rohan-helpshift) [inline docs/pc-and-console/helpshift-apis.mdx:210]: `New Conversation`

Code formatted since it is a CTA.

**Review by rohan-helpshift — CHANGES_REQUESTED** [review]

File: docs/pc-and-console/helpshift-apis.mdx (hunk 1 above)
Comment (saga-hs) [reply docs/pc-and-console/helpshift-apis.mdx]: Done

File: docs/pc-and-console/helpshift-apis.mdx (hunk 2 above)
Comment (saga-hs) [reply docs/pc-and-console/helpshift-apis.mdx]: done

File: docs/pc-and-console/helpshift-apis.mdx (hunk 1 above)
Comment (saga-hs) [reply docs/pc-and-console/helpshift-apis.mdx]: Done

**Review by rohan-helpshift — APPROVED** [review]

**Review by oitihjya-hs — APPROVED** [review]

**Review by namanKumar08 — APPROVED** [review]

**Pull request merged**